src/main/index.ts

```diff
@@ -391,6 +403,15 @@ async function createWindow() {
       mainWindow!.setSkipTaskbar(true)
       event.preventDefault()
     })
+
+    // Minimize to tray immediately if "hide on launch" selected
+    // or if --hidden arg is passed
+    if ((generalPreferences.other.hideOnLaunch || args.hidden) && !args.show) {
+      mainWindow.once('show', () => {
+        mainWindow!.hide()
+        mainWindow!.setSkipTaskbar(true)
+      })
+    }
```

Comment:
We can use this feature only for Windows and Linux, right?

Reply:
@h3poteto Correct. This feature likely wouldn't work properly on macOS due to the window handling differences. As such, I placed the additional setting in the *Other options* section which is hidden on macOS.

![image](https://user-images.githubusercontent.com/10586436/211052292-4c4667ac-b3ba-4e86-9046-5581966b223e.png)

The code on [L409-414](https://github.com/h3poteto/whalebird-desktop/blob/763006f06b77b4a115548824aaba3dc5cc356110/src/main/index.ts#L409-L414) which acts on the setting is also in the conditional block excluding macOS ([L382](https://github.com/h3poteto/whalebird-desktop/blob/763006f06b77b4a115548824aaba3dc5cc356110/src/main/index.ts#L382)), so will never run on that platform, regardless of command line parameters or manual editing of the config files.

Reply:
OK. Then please pass the CI.

Reply:
I've changed the non-null assertions to null-conditional ones, which should now pass the eslint checks.